generators/ruby-v2/model/src/ModelGeneratorContext.ts

```diff
@@ -45,7 +45,15 @@ export class ModelGeneratorContext extends AbstractRubyGeneratorContext<ModelCus
 
     public getCoreAsIsFiles(): string[] {
         const files = [
-            // Errors
+            // Public errors
+            AsIsFiles.ApiError,
+            AsIsFiles.ClientError,
+            AsIsFiles.RedirectError,
+            AsIsFiles.ResponseError,
+            AsIsFiles.ServerError,
+            AsIsFiles.TimeoutError,
```

Comment:
@dsinghvi unsure if these changes are right since I don't know what model generation is for, please let me know if this is wrong.